### PR TITLE
Add a brand to MetadataContext so it will never fall victim to duck typing

### PR DESCRIPTION
**Pull request by patrick-codaio**
Context: https://codaproject.slack.com/archives/C01835XEBSS/p1670975809120289?thread_ts=1667494841.972689&cid=C01835XEBSS

### CODE REVIEW

File: api.ts [hunk 1]
```diff
@@ -1220,12 +1220,13 @@ export function makeMetadataFormula(
...
     // regular pack functions (which this is)
     execute([search, serializedFormulaContext], context) {
-      let formulaContext = {};
+      let formulaContext: any = {};
       try {
         formulaContext = JSON.parse(serializedFormulaContext);
       } catch (err: any) {
         //  Ignore.
       }
+      formulaContext.__brand = 'MetadataContext';
```
Comment (jonathan-codaio) [inline api.ts:1229]: So the id with brands is that they should only exist within the type system and not affect anything at runtime. Usually you'd do this with a cast.

Here I think you can just declare `let formulaContext = {} as MetadataContext;`

File: requirements.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@
 bracex==2.3.post1 ; python_version >= '3.7'
 cairocffi==1.4.0 ; python_version >= '3.7'
 cairosvg==2.5.2
-certifi==2022.12.7 ; python_version >= '3.6'
+certifi==2022.9.24 ; python_version >= '3.6'
```
Comment (jonathan-codaio) [inline requirements.txt:7]: Hmm weird, this looks like it regressed. Would just revert this file.

**Review by jonathan-codaio — COMMENTED** [review]
Nice. Does this work ok on the `coda` side?

**Review by huayang-codaio — COMMENTED** [review]
does this PR only change some type definition? How would it fix the issue?

**Comment by huayang-codaio** [comment]
let me read the code further.

**Comment by patrick-codaio** [comment]
@huayang-codaio The actual fix will be in the Coda repo after I can use this typing update.

File: requirements.txt (hunk 2 above)
Comment (patrick-codaio) [reply requirements.txt]: Done

File: api.ts (hunk 1 above)
Comment (patrick-codaio) [reply api.ts]: Done, and testing in coda repo now...

**Review by huayang-codaio — APPROVED** [review]

**Comment by patrick-codaio** [comment]
@jonathan-codaio Verified the fix in coda repo: https://github.com/coda/coda/pull/83618

**Review by jonathan-codaio — APPROVED** [review]

**Pull request merged**